src/transformers/models/albert/modeling_tf_albert.py

```diff
@@ -818,6 +819,32 @@ def __init__(self, config, *inputs, **kwargs):
     def get_output_embeddings(self):
         return self.albert.embeddings
 
+    def resize_token_embeddings(self, new_num_tokens):
```

Comment:
In PT, albert can just use the "normal" `resize_token_embeddings` function, see: https://github.com/huggingface/transformers/pull/8880/files#r534143307

Reply:
Do we need a special case here?

Reply:
If we don't do that, the two bias are not resized, so yes it is mandatory.

Reply:
Maybe a comment on why it's necessary would help (I remember asking you too why there were two biases in ALBERT ;-) )

Reply:
Take a look, there is already a comment ;)
```
# ALBERT is a special case where there are two bias to update
# even though self.bias is not used anywhere and is here
# just to make the loading weights from a PT model happy
```

Reply:
Ah sorry I was responding to fast here and didn't look closely. That's great then :-)